erpnext/accounts/doctype/loyalty_program/test_loyalty_program.py

```diff
@@ -113,7 +113,7 @@ def test_cancel_sales_invoice(self):
 	def test_sales_invoice_return(self):
 		frappe.db.set_value("Customer", "Test Loyalty Customer", "loyalty_program", "Test Single Loyalty")
 		# create a new sales invoice
-		si_original = create_sales_invoice_record(2)
+		si_original = create_sales_invoice_record(200)
```

Comment:
why are we increasing the quantity? Rest all looks good

Reply:
Because using current quantity the amount of sales invoice does't fall loyalty program as i told you earlier code was broke on calculation of points from sales amount